pkg/server/server_operation_test.go

```diff
@@ -0,0 +1,83 @@
+package server
+
+import (
+	"github.com/TBD54566975/ssi-sdk/credential/exchange"
+	"github.com/goccy/go-json"
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	"github.com/stretchr/testify/assert"
+	"github.com/tbd54566975/ssi-service/pkg/server/router"
+	"github.com/tbd54566975/ssi-service/pkg/service/operation"
+	"github.com/tbd54566975/ssi-service/pkg/storage"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+)
+
+func TestOperationsAPI(t *testing.T) {
+	t.Run("GetOperations", func(t *testing.T) {
+		t.Run("Returns empty when no operations stored", func(t *testing.T) {
```

Comment:
nit: on the inner runners you can use different variables than `t` to not shadow the parent t's

Reply:
I'm of the opinion that shadowing makes sense here, unless there is a strong reason not to do so. If you find yourself having to access the outer `t` from inside a subtest, that's an indication that there is something unusual happening. Curious what your thoughts are?

Reply:
have run into shadowing errs with unexpected state, especially when tests are run in parallel. IMO as much as possible we should make our tests be able to run in parallel, which means eliminating shared state. The test as written is fine - so no problem.